### ISSUE TITLE
[cli] Integration test failures for CLI on master.

### ISSUE THREAD

**Issue by usergenic** [Polymer/tools#378]
```
  1) polymer shop the 3.0 branch when built with polymer build works in the es6-bundled configuration:
     Error: Error during rAFs after loading http://127.0.0.1:8081/ in Chrome. Browser Error:
Error: SyntaxError: Cannot use 'import.meta' outside a module
      at lib/test/integration/integration_test.js:88:19
      at Generator.next (<anonymous>)
      at fulfilled (lib/test/integration/integration_test.js:13:58)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:182:7)

  2) polymer shop "after each" hook for "works in the es6-bundled configuration":
     Error: Error encountered in browser page while testing: Error: SyntaxError: Cannot use 'import.meta' outside a module
      at Suite.<anonymous> (lib/test/integration/integration_test.js:447:19)
      at Generator.next (<anonymous>)
      at lib/test/integration/integration_test.js:16:71
      at new Promise (<anonymous>)
      at __awaiter (lib/test/integration/integration_test.js:12:12)
      at Context.teardown (lib/test/integration/integration_test.js:445:20)
```

**Comment by usergenic** [comment]
Fixed in #380.

**Comment by ranjeet-choudhary** [comment]
"polymer-cli": "^1.7.7",
getting the below error on Opera 54 with es5-bundled. 
esm-bundled works fine.

![image](https://user-images.githubusercontent.com/13473684/43048853-2eacac44-8e0c-11e8-818c-4ed2f65c4d29.png)
